tests/integration/settled-test.js

```diff
@@ -144,6 +146,17 @@ module('settled real-world scenarios', function(hooks) {
     await teardownContext(this);
   });
 
+  test('basic behavior', async function(assert) {
+    await settled();
+
+    assert.ok(
+      isSettled(),
+      `out of balance xhr semaphores are cleaned up on teardown: ${JSON.stringify(
```

Comment:
Can you tweak this message (it really isn't about semaphores here)...

Reply:
Oops! Bad copy paste :)